src/openms/include/OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/IsoSpecWrapper.h

```diff
@@ -236,14 +241,17 @@ namespace OpenMS
       **/
     IsoSpecTotalProbGeneratorWrapper(const EmpiricalFormula& formula, double p);
 
-    inline bool nextConf() final { return ILG.advanceToNextConfiguration(); };
-    inline Peak1D getConf() final { return Peak1D(ILG.mass(), ILG.prob()); };
-    inline double getMass() final { return ILG.mass(); };
-    inline double getIntensity() final { return ILG.prob(); };
-    inline double getLogIntensity() final { return ILG.lprob(); };
+    ~IsoSpecTotalProbGeneratorWrapper();
+
+    bool nextConf() final;
+    Peak1D getConf() final;
+    double getMass() final;
+    double getIntensity() final;
+    double getLogIntensity() final;
 
 protected:
-    IsoSpec::IsoLayeredGenerator ILG;
+    IsoSpecTotalProbGeneratorWrapper(const IsoSpecTotalProbGeneratorWrapper&) = delete;
```

Comment:
nitpicking here but:
would it make sense to move the copy ctor up into the area of all other ctors?! Just to have them all in one place, in case someone is checking them and sees that this one seems "missing"?